### DISCUSSION TITLE
Heiman HS2WD-E siren triggering with Sonoff ZBBridge

### DISCUSSION THREAD

**Discussion by krzysztof-lorenc** [arendst/Tasmota#10721] [Q&A] [answered]
Does anyone had success with triggering siren on [Heiman HS2WD-E](https://www.zigbee2mqtt.io/devices/HS2WD-E.html) using console on tasmotized ZbBridge?
I've tried many ZbSend command variations like for example:

`ZBSend {"Device":"0x5BD2", "send": "warning":{"duration": 10, "mode": "emergency", "strobe": false}}`
The response for this one is:
`{"ZbResponse":{"Device":"0x5BD2","Command":"FFFF!00","Status":195,"StatusMessage":"UNSUPPORTED_CLUSTER","Endpoint":1,"LinkQuality":26}`

If I set the endpoint to 2 like:
`ZBSend {"Device":"0x5BD2", "endpoint": 2, "send": "warning":{"duration": 10, "mode": "emergency", "strobe": false}}`
I get:
`{"ZbSend":"Done"}`
but the siren is not activated. Maybe I should set the endpoint or/and cluster but to what values?

**Comment by ascillato** [comment, marked as answer]
Please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). The chat is a better and more dynamic channel for helping you.

Thanks.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
